modules/claims_api/lib/custom_error.rb

```diff
@@ -36,13 +37,12 @@ def raise_backend_exception(key, source, error = nil)
     def get_source
       if (@error.respond_to?(:key) && @error.key.present?) ||
          (@error.respond_to?(:backtrace) && @error.backtrace.present?)
-        matches = if @error.backtrace.nil?
+        matches = if @error.backtrace.nil? && @error.key.present?
```

Comment:
@stiehlrod  This could just be my fake data so really just looking for a confirmation and perhaps an example error to use, but when I send a test error into this the `key` value is located inside `original_message` (`@error.original_message`).  Do you have an example error that you are using when testing this that I can try to grab?